src/messaging/messaging-types.ts

```diff
@@ -880,3 +844,28 @@ function validateAndroidFcmOptions(fcmOptions: AndroidFcmOptions) {
       MessagingClientErrorCode.INVALID_PAYLOAD, 'analyticsLabel must be a string value');
   }
 }
+
+/**
+ * Transforms milliseconds to the format expected by FCM service.
+ * Returns the duration in seconds with up to nine fractional
+ * digits, terminated by 's'. Example: "3.5s".
+ *
+ * @param {number} milliseconds The duration in milliseconds.
+ * @return {string} The resulting formatted string in seconds with up to nine fractional
+ * digits, terminated by 's'.
+ */
+export function transformMillisecondsToSecondsString(milliseconds: number): string {
```

Comment:
Doesn't need to be exported (I think). In general be conservative about what's exported from each module.

Reply:
Good catch! This was copypasta when the function was moved from utils. 
Thanks! Fixed.